levant/templater.go

```diff
@@ -43,9 +52,7 @@ func RenderTemplate(templateFile, variableFile string, flagVars *map[string]stri
 		tpl, err = renderYAMLVarsTemplate(string(src), variableFile, flagVars)
 	case "":
 		if len(*flagVars) == 0 {
-			logging.Debug("levant/template: no variables file or var flags, skipping templating")
 			tpl = bytes.NewBuffer(src)
-			break
```

Comment:
Removed this break as it would skip the template call which is necessary to catch any missing interpolations